interpreter/value_pathcapability.go

```diff
@@ -180,6 +180,14 @@ func (v *PathCapabilityValue) GetMember(context MemberAccessibleContext, locatio
 	return nil
 }
 
+func (v *PathCapabilityValue) GetMethod(
+	_ MemberAccessibleContext,
+	_ LocationRange,
+	_ string,
+) FunctionValue {
+	return nil
```

Comment:
Do the `borrow` and `check` functions from `GetMember` have to be moved here?

Reply:
I left these out since PathCapability values are deprecated. Anywa added them in https://github.com/onflow/cadence/pull/3886/commits/51ad86e22d8fd774fc1200158e426767cb57f28f for completion.